packages/components/src/Text/Text.tsx

```diff
@@ -16,7 +16,7 @@ type Props = {
   spacing?: TypographyProps<TextElement>["spacing"];
   tabIndex?: number;
   weight?: TypographyProps<TextElement>["weight"];
-};
+} & React.HTMLAttributes<HTMLElement>;
```

Comment:
Is this mostly so it has access to things like `id`, `hidden`, `style`, etc?

Reply:
Yea, was trying to use this as a target for `aria-labelledby`